doc/user_manual/generated/generateRomDoc.py

```diff
@@ -748,6 +750,7 @@
               'Clusters',
               'Interpolated']
 validDNNRom = ['KerasMLPClassifier',
+              'KerasMLPRegression'
```

Comment:
There is a missing comma here, the internalRom.tex is still showing the KerasMLPRegression manual. Could you update it and regenerate the internalRom.tex? @wanghy-anl